pkgs/development/python-modules/pdfkit/default.nix

```diff
@@ -5,12 +5,12 @@
 
 buildPythonPackage rec {
   pname = "pdfkit";
-  version = "0.5.0";
+  version = "0.6.1";
 
   src = fetchPypi {
     inherit pname version;
-    extension = "zip";
-    sha256 = "1p1m6gp51ql3wzjs2iwds8sc3hg1i48yysii9inrky6qc3s6q5vf";
+    extension = "tar.gz";
```

Comment:
this is the default case, you can remove
```suggestion
```

Reply:
right